src/signals/incident-management/containers/IncidentSplitContainer/styled.js

```diff
@@ -73,8 +73,34 @@ export const StyledLabel = styled(Label)`
   }
 `
 
+export const StyledMainContainer = styled.div`
+  display: grid;
+  grid-column: span 7;
+  row-gap: ${themeSpacing(8)};
+
+  @media screen and ${breakpoint('max-width', 'laptopM')} {
+    grid-column: span 10;
+  }
+
+  @media screen and ${breakpoint('max-width', 'laptop')} {
+    grid-column: span 12;
+  }
+`
+
+export const StyledButtonContainer = styled.div`
+  grid-column: span 7;
+`
+
 export const StyledFieldset = styled.fieldset`
+  background-color: ${themeColor('tint', 'level3')};
   scroll-margin-top: ${themeSpacing(15)};
+  grid-column: span 12;
+  padding: ${themeSpacing(5)};
+  margin-inline: -${themeSpacing(5)};
```

Comment:
This is a negative margin, which isn't used by SIgnalen.

Reply:
Do we use another pattern to allow for background color overflow?